### ISSUE TITLE
Unmanaged memory warnings in margin cache import

### ISSUE THREAD

**Issue by smcguire-cmu** [astronomy-commons/hats-import#267]
When running the margin cache import, there are a lot of repeated dask errors like this: `distributed.worker.memory - WARNING - Unmanaged memory use is high. This may indicate a memory leak or the memory may not be released to the OS; see https://distributed.dask.org/en/latest/worker-memory.html#memory-not-released-back-to-the-os for more information.`

An example of this is in this notebook https://lsdb--166.org.readthedocs.build/en/166/notebooks/des-gaia.html

**Comment by delucchi-cmu** [comment]
I'm not seeing errors like this, using the most recent version of the notebook and hats/hats-import. Is this still an issue?

**Comment by nevencaplar** [comment]
@smcguire-cmu when you get a chance, can you please check if this is still an issue.

**Comment by smcguire-cmu** [comment]
I don't think it is, at least I don't see the errors any more when running the notebook either and can't recreate it. so I think we can close this one out.
